cookbooks/aws-parallelcluster-awsbatch/recipes/install.rb

```diff
@@ -45,7 +45,7 @@
       curl --retry 3 -L -o aws-parallelcluster.tgz ${custom_package_url}
       mkdir aws-parallelcluster-awsbatch-cli
       tar -xzf aws-parallelcluster.tgz --directory aws-parallelcluster-awsbatch-cli
-      cd aws-parallelcluster-awsbatch-cli/*aws-parallelcluster-*
```

Comment:
Why is this change?

Reply:
because when the folder was names `aws-parallelcluster` before it was zipped, then it would not match the pattern.